scripts/tests/create_ios_app_store_profile_test.rb

```
@@ -0,0 +1,273 @@
+# credentials or network access are used. Run with:
+#
+#   ruby scripts/tests/create_ios_app_store_profile_test.rb
+
+require "minitest/autorun"
+require_relative "../create-ios-app-store-profile"
+
+module IOSProfileBootstrap
+  # Scripted transport: each expectation is [method, path matcher, status,
+  # body]. Requests are matched in order; an unexpected request fails the test.
+  class FakeTransport
+    Call = Struct.new(:method, :path, :body)
```

RuboCop (1.88.2)

[warning] `:method` member overrides `Struct#method` and it may be unexpected.

(Lint/StructNewOverride)